mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnC.tsx

```diff
@@ -0,0 +1,47 @@
+import { ThemedScrollView, ThemedText } from '@components/themed'
+import { AccordionContent, WalletAccordion } from '@components/WalletAccordion'
+import { StackScreenProps } from '@react-navigation/stack'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import { LoanParamList } from '../LoansNavigator'
+import { useTnC } from './LoansTnCText'
+
+type Props = StackScreenProps<LoanParamList, 'LoansTnC'>
+
+export function LoansTnC ({ route }: Props): JSX.Element {
```

Comment:
Do we need this component, or could we just redirect the user to the LoansFaq?

Reply:
better safe than sorry !

…also LoansFaq opens a different dropdown by default, explaining the the question that it routes from.